Error in log file for widgets without items

[This recent change](https://github.com/openhab/openhab/commit/4a89370ce66b161179a7c69c6b41ae67f01e731a#diff-cdd2dbc900e7ba12da0fd9ba4cf68b14R228) always logs an error for widgets without defined items, e.g. Frames, because the widget.getItem() returns null.
For example, this sitemap definition
```
Frame label="Licht" {
...
}
```
results in 
```
22:12:23.568 [ERROR] [o.u.i.items.ItemUIRegistryImpl:228  ] - Cannot retrieve item null for widget org.openhab.model.sitemap.Frame
```

This is probably also the reason for pull request #2615 :-)

--- a/bundles/ui/org.openhab.ui/src/main/java/org/openhab/ui/internal/items/ItemUIRegistryImpl.java
+++ b/bundles/ui/org.openhab.ui/src/main/java/org/openhab/ui/internal/items/ItemUIRegistryImpl.java
@@ -225,7 +225,7 @@ public class ItemUIRegistryImpl implements ItemUIRegistry {
 		try {
 			item = getItem(itemName);
 		} catch (ItemNotFoundException e) {
-			logger.error("Cannot retrieve item {} for widget {}", itemName, w.eClass().getInstanceTypeName());
+			logger.debug("Cannot retrieve item {} for widget {}", itemName, w.eClass().getInstanceTypeName());
 		}
 		return formatLabel(item, itemName, label);
 	}

